migrations/2017_04_07_114138_add_is_private_to_discussions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('discussions', function (Blueprint $table) {
+            $table->boolean('is_private', false);
+        });
+    },
+
+    'down' => function (Builder $schema) {
+        $schema->table('discussions', function (Blueprint $table) {
+            $table->dropColumn('is_private');
+        });
+    }
+];
```

Comment:
can we please use the `Migration::addColumns` shortcut, as in https://github.com/flarum/core/blob/master/migrations/2015_10_24_194000_add_ip_address_to_posts.php